src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+        }
```

Comment:
You are creating new instances of Lottery inside a loop. This is unnecessary if the Lottery class does not maintain any state that changes during the getRandomBall() calls. Consider moving the 'Lottery lottery = new Lottery();' line outside of the loop to avoid creating multiple instances.